legacy/firmware/.changelog.d/4324.added

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
drop the leading empty line please

Reply:
Removed in 8ae3a80f9c65bddf156c1b9805ff0c97f2ab618d